ReactiveCocoa/NSObject+KeyValueObserving.swift

```diff
@@ -10,15 +10,16 @@ extension Reactive where Base: NSObject {
 	///
 	/// - parameters:
 	///   - keyPath: The key path of the property to be observed.
+	///   - options: KVO options, see `NSKeyValueObservingOptions` for more documentation.
 	///
 	/// - returns:
 	///   A producer emitting values of the property specified by the key path.
-	public func values(forKeyPath keyPath: String) -> SignalProducer<Any?, NoError> {
+	public func values(forKeyPath keyPath: String, options: NSKeyValueObservingOptions = [.initial, .new]) -> SignalProducer<Any?, NoError> {
```

Comment:
It should not be exposed, since given the API, `old` and `prior` would not be usable at all.